src/app/client/src/app/modules/core/services/user/user.service.ts

```diff
@@ -247,10 +247,13 @@ export class UserService {
     };
     this.publicDataService.post(option).subscribe
       ((data: ServerResponse) => {
+        const orgDetails = {};
         _.forEach(data.result.response.content, (orgData) => {
           this.orgNames.push(orgData.orgName);
+          orgDetails[orgData.identifier] = orgData.orgName;
         });
         this._userProfile.organisationNames = this.orgNames;
+        this._userProfile.orgDetails = orgDetails;
```

Comment:
orgDetails should contain all the orgDetails add getter method to get required data in required format